packages/api/src/utility.ts

```diff
@@ -55,7 +55,7 @@ export function mapSubscriptionsAsCsv(
       'paymentMethodID',
       'deactivationDate',
       'deactivationReason'
-    ].join(',') + '\n'
+    ].join(';') + '\n'
```

Comment:
Same objection as described in in https://github.com/wepublish/wepublish/pull/767/files#r1022846852

> Shouldn't we be RFC compliant https://www.ietf.org/rfc/rfc4180.txt? Since CSV stands for: Comma-Separated Values Escape commas?

Reply:
solved